.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+---
+blank_issues_enabled: false
+contact_links:
+  - name: Kubesec Discussions
+    url: https://github.com/controlplaneio/kubesec/discussions
+    about: Please ask and answer questions here!
+# not required, auto picks up SECURITY.md
+#   - name: Security
+#     url: X
+#     about: Please report security vulnerabilities here.
```

Comment:
Is it worth to remove the comment? Or is this considered documentation?

Reply:
Yeah I added this in-case we look at this file in the future and wonder why it's missing.